### PR TITLE
Change the .service class behavior

### PR DESCRIPTION
**Pull request by mldk-tech**
#96

### CODE REVIEW

**Comment by mldk-tech** [comment]
You can see just the hover. 
i remove the css animation.

bar
bar red

**Comment by mldk-tech** [comment]
You can see that the icon color in white
color

**Comment by niryariv** [comment]
yes, much better in white. thanks!

File: ibli/themes/custom/bootstrap_subtheme/css/style.css [hunk 1]
```diff
@@ -5107,25 +5104,13 @@ button.close {
       font-size: 30px;
       display: block; }
     .services ul li:hover {
-      width: 328px !important;
       background: #69D2E7;
-      cursor: pointer;
```
Comment (NofarGizra) [inline ibli/themes/custom/bootstrap_subtheme/css/style.css:5112]: Keep the `cursor:pointer` line.

**Comment by NofarGizra** [comment]
Merged.

**Pull request closed without merging**